### ISSUE TITLE
Duplicated routes when using prefix.

### ISSUE THREAD

**Issue by ciacka** [PavelPolyakov/fastify-blipp#2]
**The problem**

List of routes are duplicated for the same HTTP method, when using Fastify's `register` function with `prefix` option in some specific cases.

**Example**

```javascript
// module.js
export default async (fastify) => {
  fastify.get('/', () => {
    return [];
  });
};

// app.js
import module from './module';

// ...

fastify.register(module, { prefix: '/products' });
```

**Blipp output**

```bash
🏷️  Routes:
GET	/products/
GET	/products/
```

**Expected behavior**
The Blipp output should be as following:
```bash
🏷️  Routes:
GET	/products
```

**Exception**
When code in the `module.js` file is changed to the following (path is changed to `/index` from `/`):
```javascript
// module.js
export default async (fastify) => {
  fastify.get('/index', () => {
    return [];
  });
};
```

Then the Blipp output is as the following:
```bash
🏷️  Routes:
GET	/products/index
```

**Comment by PavelPolyakov** [comment]
hi @ciacka , thanks for reporting.
I'll, probably, have a look on this next week. Feel free to submit PR if you'd like to contribute.

**Comment by PavelPolyakov** [comment]
I've just released `fastify-blipp@2.0.0` where prefixed routes are handled more correctly.

There still will be a "duplication" by default, but this duplication is how the prefixed routes are handled by default:
https://www.fastify.io/docs/latest/Routes/#handling-of-route-inside-prefixed-plugins

Therefore I'm closing this issue.
